redaxo/src/core/boot.php

```diff
@@ -120,6 +120,21 @@
     }
 }
 
+// ----------------- HTTPS REDIRECT
+if (rex::getProperty('use_https') === true ||
+    (rex::getProperty('use_https') === 'frontend' && !rex::isBackend()) ||
+    (rex::getProperty('use_https') === 'backend' && rex::isBackend())) {
```

Comment:
Das ließe sich abkürzen zu 

```
rex::getProperty('use_https') === true || rex::getProperty('use_https') === rex::getEnvironment()
```

So wäre auch gleich sichergestellt, dass in der Console nichts davon ausgeführt wird, denn in der Console liefert `getEnvironment` "console", `isBackend` ist aber trotzdem `true`.